scripts/segmentation_HALO-20240912a.md

```diff
@@ -0,0 +1,411 @@
+* use the list of `remarks` to state any deviations, also with respective times
+
+Alternatively, you can also define the segments as dictionaries which also allows to add further attributes to single segments, e.g. a `radius` to a `circle` segment. At the end of the following code block all segments will be normalized by the `parse_segments` function.
+
+```python
+seg1 = (
+    slice("2024-09-12T11:33:02", "2024-09-12T11:46:57"),
+    ["straight_leg", "ascent"],
+    "eastward ascent after takeoff",
+)
+
+seg2 = (
+    slice("2024-09-12T11:52:44", "2024-09-12T12:01:28"),
+    ["straight_leg", "ascent"],
+    "eastward ascending ferry",
+
+)
+
+seg3 = (
+    slice("2024-09-12T12:01:28", "2024-09-12T12:35:09"),
+    ["straight_leg"],
+    "long eastward ferry",
+)
+
+seg4 = (
+    slice("2024--09-12T12:36:26", "2024-09-12T13:02:59"),
+    ["straight_leg"],
+    "leg to first circle"
+)
+
+seg5 = (
+    slice("2024-09-12T13:06:53", "2024-09-12T14:02:18"),
+    ["circle"],
+    "clockwise eastern circle",
+)
+
+seg6 = (
+    slice("2024-09-12T14:06:42", "2024-09-12T14:12:06"),
+    ["straight_leg"],
+    "leg to second circle",
+    ["irregularity: roll angle deviation up to plus/minus 4 degree"],
+)
+
+seg7 = (
+    slice("2024-09-12T14:18:20", "2024-09-12T15:10:37"),
+    ["circle"],
+    "counterclockwise north-eastern circle",
+)
+
+seg8 = (
+    slice("2024-09-12T15:13:08", "2024-09-12T15:18:19"),
+    ["straight_leg"],
+    "leg through circle",
+)
+
+seg9 = (
+    slice("2024-09-12T15:22:30", "2024-09-12T15:45:40"),
+    ["straight_leg"],
+    "leg to third circle",
+)
+
+# seg10 = (
+#    slice("2024-09-12T15:46:03", "2024-09-12T15:51:48"),
+#    ["circle"],
+#    "counterclockwise holding pattern",
+#    ["small waiting circle"]
+# )
+
+seg11 = (
+    slice("2024-09-12T16:03:30", "2024-09-12T17:03:44"),
+    ["circle"],
+    "counterclockwise north-western circle",
+)
+
+seg12 = (
+    slice("2024-09-12T17:12:26", "2024-09-12T17:53:12"),
+    ["straight_leg", "ec_track"],
+    "leg on EC track",
+    ["contains EC underpass"],
+)
+
+seg13 = (
+    slice("2024-09-12T17:56:13", "2024-09-12T18:55:00"),
+    ["circle"],
+    "clockwise western circle",
+)
+
+seg14 = (
+    slice("2024-09-12T18:56:07", "2024-09-12T19:41:13"),
+    ["straight_leg"],
+    "long westward ferry"
+)
+
+seg15 = (
+    slice("2024-09-12T19:41:13", "2024-09-12T19:45:16"),
+    ["straight_leg", "descent"],
+    "westward descending ferry",
+)
+
+seg16 = (
+    slice("2024-09-12T19:46:30", "2024-09-12T19:58:55"),
+    ["straight_leg", "descent"],
+    "westward descending ferry"
+)
+
+seg17 = (
+    slice("2024-09-12T20:01:29", "2024-09-12T20:05:05"),
+    ["straight_leg", "descent"],
+    "final descent",
+)
+
+
+# add all segments that you want to save to a yaml file later to the below list
+segments = [parse_segment(s) for s in [seg1, seg2, seg3, seg4, seg5, seg6, seg7, seg8, seg9, seg10, seg11, seg12, seg13, seg14, seg15, seg16, seg17]]
```

Comment:
```suggestion
segments = [parse_segment(s) for s in [seg1, seg2, seg3, seg4, seg5, seg6, seg7, seg8, seg9, seg11, seg12, seg13, seg14, seg15, seg16, seg17]] #seg10 removed because currently commented out
```